src/database/DataTypes/Name.js

```diff
@@ -149,6 +149,7 @@ Name.schema = {
     policies: { type: 'linkingObjects', objectType: 'InsurancePolicy', property: 'patient' },
     female: { type: 'bool', default: false },
     thisStoresPatient: { type: 'bool', default: false },
+    nameTagJoins: { type: 'linkingObjects', objectType: 'NameTagJoin', property: 'name' },
```

Comment:
Same as a `list`, just automatically handled by `realm` technically don't need the join table. Would be happy to not use it, but we seem to like using them and I'm confused and trying to be consistent

Reply:
Oh yea, meta data. None present in this case, but who knows what the future holds. Consistent to keep join tables as we have in the other cases (and necessarily due to meta data).